modules/tinymce/src/themes/silver/main/ts/ui/toolbar/CommonToolbar.ts

```diff
@@ -199,7 +201,8 @@ const renderSlidingMoreToolbar = (toolbarSpec: MoreDrawerToolbarSpec): SketchSpe
     onClosed: (comp) => {
       // TINY-9223: This will only broadcast to the same mothership as the toolbar
       comp.getSystem().broadcastOn([ Channels.toolbarHeightChange() ], { type: 'closed' });
-    }
+    },
+    onToggled: toolbarSpec.onToggled,
```

Comment:
It's pretty minor, but I'm not sure "toggling" a toolbar would mean expanding / shrinking it to me. It seems like it would be more about showing/hiding. Can we make this configuration name more descriptive? I don't have any better names by the way :)

Reply:
Don't know about that one. In case of the sliding toolbar maybe it makes sense, but for the floating it's exactly like toggling, no? 

We can reword it as `onMoreButtonClick`. However it's verbose and do not really underline the intentions. I'd prefer to leave `onToggled` here.

Reply:
Yeah, as I said, I don't have any better ideas.